src/classes/HH_Container_LCTRL.cls

```diff
@@ -115,6 +142,7 @@ public class HH_Container_LCTRL {
             if (hhId == null)
                 return null;
 
+			// conSelect does the FLS check
```

Comment:
agreed.  I copied and modified this from some older code in NPSP.  I'll update it to a function.